models/social/arbitrum/social_arbitrum_schema.yml

```diff
@@ -0,0 +1,139 @@
+version: 2
+
+models:
+  - name: cipher_arbitrum_base_trades
+    meta:
+      blockchain: arbitrum
+      sector: social
+      project: cipher
+      contributors: hildobby
+    config:
+      tags: ['table', 'social', 'arbitrum', 'trades', 'cipher']
+    description: "Cipher basic trades info"
```

Comment:
once you set unique keys for incremental, be sure to add test here (and all models that need it)